src/libs/API/parameters/ChangeTransactionsReportParams.ts

```diff
@@ -0,0 +1,6 @@
+type ChangeTransactionsReportParams = {
+    transactionList: string;
+    reportID: string;
+    reportActionIDToThreadReportIDMap: Record<string, string>; // A map linking the optimistic MOVEDTRANSACTION or UNREPORTEDTRANSACTION reportActionID to the transaction thread reportID.
+};
```

Comment:
We renamed this param to `transactionIDToReportActionAndThreadData`

```suggestion
// A map linking the optimistic MOVEDTRANSACTION or UNREPORTEDTRANSACTION reportActionID to the transactionID. If we're creating the transactionThread as part of moving the transaction, we should also send the optimistic transactionThreadReportID and transactionThreadCreatedReportActionID
type TransactionThreadInfo = {
    transactionThreadReportID?: string;
    transactionThreadCreatedReportActionID?: string;
    movedReportActionID: string;
};

type ChangeTransactionsReportParams = {
    transactionList: string;
    reportID: string;
    transactionIDToReportActionAndThreadData = Record<string, TransactionThreadInfo>; // A map of transactionID to TransactionThreadInfo
};
```

Reply:
Thanks for this clarification.